test/Microsoft.SqlTools.ServiceLayer.PerfTests/Tests/Common.cs

```diff
@@ -12,7 +12,7 @@
 using Microsoft.SqlTools.ServiceLayer.Workspace.Contracts;
 using Xunit;
 
-namespace Microsoft.SqlTools.ServiceLayer.PerfTests.Tests
+namespace Microsoft.SqlTools.ServiceLayer.PerfTests
```

Comment:
I fixed the namespace in the project

---
In reply to: [88957713](https://github.com/Microsoft/sqltoolsservice/pull/155#discussion_r88957713) [](ancestors = 88957713)